File: templates/python/es/text/code/pothole.py
from karel.stanfordkarel import *
def main():
   moverse()
   llenar_baches()
   moverse()

# Rellena el bache debajo de la posición actual de Karel
# colocando un cono en esa esquina. Para que esta función funcione
# correctamente, Karel debe estar orientado hacia el este inmediatamente arriba del
# bache. Cuando se complete la ejecución, Karel tendrá
# regresó al mismo cuadrado y nuevamente estará orientado hacia el este.
def llenar_baches():
   girar_derecha()
   moverse()
   poner_cono()
   media_vuelta()
   moverse()
   girar_derecha()

# Gira Karel 90 grados a la derecha.
def girar_derecha():
   girar_izquierda()
   girar_izquierda()
   girar_izquierda()

# Gira Karel alrededor de 180 grados.
def media_vuelta():
   girar_izquierda()
   girar_izquierda()

File: templates/java/es/text/code/pothole.java
import stanford.karel.*;
public class RellenoDeBachesKarel extends Karel {
   public void run() {
      moverse();
      llenarBaches();
      moverse();
   }

   /**
    * Rellena el bache por debajo Karel posición actual
    * 's mediante la colocación de un cono en esa esquin
    * a. Para que este método funcione correctamente, Ka
    * rel debe estar orientado hacia el este inmediatame
    * nte por encima del bache. Cuando se complete la ej
    * ecución, Karel habrán regresado a la misma plaza y
    * nuevamente estarán orientados hacia el este.
    */
   private void llenarBaches() {
      girarDerecha();
      moverse();
      ponerCono();
      mediaVeulta();
      moverse();
      girarDerecha();
   } 

   /** Gira Karel 90 grados a la derecha. */
   public void girarDerecha() {
      girarIzquierda();
      girarIzquierda();
      girarIzquierda();
   }

   /** Gira Karel alrededor de 180 grados. */
   public void mediaVeulta() {
      girarIzquierda();
      girarIzquierda();
   }
}